src/test/ui/lint/forbid-group-group-1.rs

```diff
@@ -0,0 +1,13 @@
+// Check what happens when we forbid a smaller group but
+// then deny a superset of that group.
+
+#![forbid(nonstandard_style)]
+
+// FIXME: Arguably this should be an error, but the WARNINGS group is
```

Comment:
does this mean you should have a different test that tries this subset/superset case using something *other* than the special cased WARNINGS group?

(or is the WARNINGS group the only instance of a superset in the first place...?)

Reply:
I couldn't find any other group to use.